janus_server/src/aggregator.rs

```diff
@@ -185,6 +198,16 @@ impl<C: Clock> Aggregator<C> {
     }
 }
 
+impl<C> Aggregator<C> {
+    /// Returns the HTTPS URL of this aggregator's own endpoint.
+    fn own_endpoint(&self) -> Url {
+        // TODO (issue #20): determine this URL endpoint from configuration
+        // TODO (issue abetterinternet/ppm-specification#209): This may no longer be needed if the
+        // requirements for the "instance" problem details member change.
+        Url::parse("https://example.com/ppm_aggregator").unwrap()
+    }
+}
```

Comment:
I think this should be re-worked on top of #37 so that the aggregator's own endpoint is obtained from task parameters.